dependencies.txt

```diff
@@ -0,0 +1,11 @@
+# list ubuntu packages needed in production, one per line
+# run time dependencies
+# ensure fully working base python3 installation
+# see: https://gist.github.com/tiran/2dec9e03c6f901814f6d1e8dad09528e
+python3.9
+python3.9-dev
```

Comment:
We shouldn't need -dev in production, it should only be needed to build packages?